cmd/osmosisd/cmd/root.go

```diff
@@ -440,7 +451,12 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 
 	genAutoCompleteCmd(rootCmd)
 
-	initRootCmd(rootCmd, encodingConfig)
+	initRootCmd(rootCmd, encodingConfig, tempApp)
+
+	// UNFORKING v2 TODO: I don't think we have an option but to implement this. With out, the sdk queries do not show up in the CLI.
```

Comment:
.

Reply:
Why would we not want to implement this? Is it because we already have osmocli that is customized and tailored to our needs?